org/views.py

```diff
@@ -137,4 +137,40 @@ def Permission_Set_List(request,org_id):
             }
             response_object.append(value)
         return Response(response_object, status.HTTP_200_OK)
-    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
\ No newline at end of file
+    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
+
+
+
+@swagger_auto_schema(
+    operation_id="edit_org",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='PUT',
+    request_body = EditOrgSerializer,
+    responses={
+        '200': set_example(responses.update_org_200),
+        '400': set_example(responses.org_not_present_400),
+        '401': set_example(responses.user_unauthorized_401),
+        '403': set_example(responses.admin_access_403),
+    }
+)
+@api_view(['PUT'])
+@permission_classes([IsAuthenticated])
+def EditOrg(request,org_id):
+    org_count = Org.objects.filter(pk=org_id).count()
+    if org_count>0:
+        org = Org.objects.get(pk=org_id)
+        user = request.user
+        group = Group.objects.get(name="Admin",org=org)
```

Comment:
What I interpret from the issue is, that anyone with "admin **permission** " should be able to access it and not necessarily the **group** named "Admin". 
Since we have given them the ability to make groups, they might as well make another group with "Admin Permission" but a different name. I think that differently named group will not be able to access this API here.
@naveennvrgup sir could you please look into this, as I might have misinterpreted the issue..?

Reply:
i am glad that u found this bug @priyanshisharma . the intended behavior should as mentioned above by @priyanshisharma .

Reply:
U hav proved that u are actually putting effort into reviewing PRs.

Reply:
@priyanshisharma ma'am , I've edited the code and am checking if the user is admin or not by the permission_set object.